js/views/file_input_view.js

```diff
@@ -301,6 +301,16 @@
         return;
       }
 
+      // Save temporary copy of file to disk.
+      const arrayBuffer = await new Promise((resolve) => {
+        const fileReader = new FileReader();
+        fileReader.onload = () => resolve(fileReader.result);
+        fileReader.readAsArrayBuffer(originalFile);
+      });
```

Comment:
It would be good to refactor this little dance to get file contents into a separate function.